controllers/createConfController.js

```diff
@@ -68,20 +71,38 @@ module.exports.createConf = async (req, res) => {
       conference = await createConfWithDay(email, conferenceDay, userTimezoneOffset)
     }
   } catch (err) {
-    req.flash('error', 'La conférence n\'a pas pu être créée. Vous pouvez réessayer.')
-    console.error('Error when creating conference', err)
-    return res.redirect('/')
+    req.flash("error", "La conférence n'a pas pu être créée. Vous pouvez réessayer.")
+    console.error("Error when creating conference", err)
+    return res.redirect("/")
   }
 
   try {
     await emailer.sendConfCreatedEmail(email, conference.phoneNumber, conference.pin, durationInMinutes, conferenceDay, conference.expiresAt, config.POLL_URL, userTimezoneOffset)
-
-    return res.redirect(urls.showConf.replace(":id", conference.id) + '#' + conference.pin)
   } catch (err) {
-    req.flash('error', 'L\'email contenant les identifiants n\'a pas pu être envoyé. Vous pouvez réessayer.')
-    console.error('Error when emailing', err)
-    return res.redirect('/')
+    req.flash("error", "L'email contenant les identifiants n'a pas pu être envoyé. Vous pouvez réessayer.")
+    console.error("Error when emailing", err)
+    return res.redirect("/")
+  }
+
+  console.log(isAcceptedEmail(email, config.EMAIL_WEB_ACCESS_WHITELIST), config.FEATURE_WEB_ACCESS)
+  if (isAcceptedEmail(email, config.EMAIL_WEB_ACCESS_WHITELIST) && config.FEATURE_WEB_ACCESS) { // check if email is in whitelist
+    try {
+      const token = encrypt(jwt.sign({ roomNumber: conference.pin} , config.SECRET, { expiresIn: (durationInMinutes || 720) * 60 }))
+      await emailer.sendConfWebAccessEmail({
+        email,
+        phoneNumber: conference.phoneNumber,
+        conferenceDay: conferenceDay,
+        url: `${config.HOSTNAME_WITH_PROTOCOL}/dashboard/${token}`,
```

Comment:
utiliser plutot urls.dahsboard, que tu as déjà créé
urls.dashboard.replace(":token", token)
```suggestion
        url: `${config.HOSTNAME_WITH_PROTOCOL}${urls.dashboard.replace(":token", token)}`,
```